tests/test_actions.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the `edx-enterprise` admin actions module.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import unittest
+
+import factory
+import mock
+
+from enterprise.actions import export_as_csv_action
+
+
+class DummyModel(object):
```

Comment:
This makes sense. However, if you have no real models making use of the functionality being tested by `test_export_as_csv_callable_fields`, why is that functionality even included? That's just dead code.